forem/forem
# Static Sections for Site Navigation - Pt2

## What type of PR is this? (check all applicable)


## Description
This PR represents the second (and last) part of implementing the [Static Sections for Nav Links RFC](https://github.com/forem/rfcs/blob/2b110df1a11942d0b83126024055aba1cd7b9fed/text/0237-static-sections-nav-links.md). In this PR, I completed the following:
- [X] Add `Default` and `Other` sections in Admin
- [X] Add `Section` radio buttons to New Nav Link and Edit Nav Link forms
- [X] Add `Default` and `Other` sections in frontend; ensure nav links appear in correct section
- [X] Adjust styling; write and/or fix related specs
- [X] Update relevant documentation (PR is [here](https://github.com/forem/admin-docs/pull/10))

### NOTE: Per the RFC, there needs to be a new task that creates necessary Default and Other links for brand-new forems. I shall accomplish this in a separate PR and link to it here once cut. 🙏🏾 

## QA Instructions, Screenshots, Recordings
1. As an admin, in the Admin Portal, navigate to `Customization => Navigation Links`.
2. Create and/or Edit some links. Check that they appear correctly in the frontend.
3. Delete a few links. Make sure they are gone from the frontend.
4. Delete all Other nav links. Check that the Other section is gone from the frontend.
5. Run all specs and ensure they pass.

### UI accessibility concerns?
I made sure to follow existing a11y conventions for links and sections, as already present in the Sidebar Nav code.

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams. **Eventually, when the entire RFC is implemented, I will update the documentation and liaise with the Community Team to add to our Change Log post.**
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

![Usain Bolt dusting the competition](https://media.giphy.com/media/vCJrwYGBQlbdS/giphy-downsized.gif)
